stack/base/before-notebook.d/10_prepare-home-config.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# If the container is start by spawner and the home is remounted.
+# The .bashrc in HOME won't be set properly.
+if [[ ! -f  /home/${NB_USER}/.bashrc ]]; then
+    cp /etc/skel/.bashrc /home/${NB_USER}/.bashrc
+fi
+
+# Set sshagent by source load-singlesshagent.sh script
+# append the command text of source to .bashrc if the script /opt/bin/load-singlesshagen.sh is present
+# and the command text is not already present in .bashrc
+if [[ -f /opt/bin/load-singlesshagent.sh ]] && ! grep -q "# Load singlesshagent on shell startup." /home/${NB_USER}/.bashrc; then
```

Comment:
I would use a variable here

```
header="# Load singlesshagent on shell startup."
```

Otherwise looks okay, LMK when you want me to test this again @unkcpz

Reply:
Yes, done!